docs/configuration/worker.md

```diff
@@ -112,6 +112,7 @@ license: |
 | celeborn.worker.shuffle.partitionSplit.min | 1m | Min size for a partition to split | 0.3.0 | celeborn.shuffle.partitionSplit.min | 
 | celeborn.worker.sortPartition.indexCache.expire | 180s | PartitionSorter's cache item expire time. | 0.4.0 |  | 
 | celeborn.worker.sortPartition.indexCache.maxWeight | 100000 | PartitionSorter's cache max weight for index buffer. | 0.4.0 |  | 
+| celeborn.worker.sortPartition.reservedMemory.enabled | true | When true, partition sorter will reserve memory configured by `celeborn.worker.sortPartition.reservedMemoryPerPartition` to allocate a block of memory for warming up while sorting a shuffle file off-heap with page cache for non-hdfs files.Otherwise, partition sorter seeks to position of each block and does not warm up for non-hdfs files. | 0.5.0 |  | 
```

Comment:
when trying to understand this PR, I found this configuration name is misleading.

the main thing we want to do here is to control whether to prefetch the original partition files to leverage the Linux PageCache mechanism to speed up the subsequent random reading of them, while the `reservedMemory` is just an additional cost of the prefetch feature.

I also see another phrase "warm up" is used in this patch, this phrase is not common to be used in IO cases, personally, I think "prefetch" is more accurate.

`celeborn.worker.sortPartition.prefetch.enabled`